apps/betterangels-backend/shelters/admin.py

```diff
@@ -118,7 +120,7 @@ def _clean_choices(self, field_name: str, model_class: Type[T]) -> list[T]:
 
         # Retrieve existing objects and their names
         existing_objects = list(model_class.objects.filter(name__in=choices))  # type: ignore[attr-defined]
-        existing_entries = {obj.name for obj in existing_objects}
+        existing_entries = {str(obj) for obj in existing_objects}
```

Comment:
Previous PR opted for name while works, it should really be str(obj) so it is more generic.